.babelrc

```diff
@@ -1,35 +1,24 @@
 {
-  presets: [
-    [
-      "env",
-      {
-        targets: {
-          node: "current",
-          browsers: "last 2 chrome versions, last 2 firefox versions"
-        },
-        modules: false
-      }
-    ],
-    "react",
-    "flow"
+  "presets": [
+    "@babel/preset-env",
```

Comment:
some new default makes it a lot leaner.
The browser definition has moved to .browserlistrc, which other tools use too (if we want to use them later).